specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/examples/HubPolicy/create.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "test-rg",
+    "workspaceName": "aml-workspace-name",
+    "policyName": "policy-1",
+    "api-version": null,
+    "body": {
+      "properties": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/test-rg/providers/Microsoft.MachineLearningServices/workspaces/aml-workspace-name/policies/policy-1",
+        "name": "policy-1",
+        "azureScope": "some_string",
```

Comment:
>some_string [](http://example.com/codeflow?start=23&length=11)

is this realistic?

Reply:
nope, all the subId and name are made up for example use only.

Reply:
Will update, good catch